### PR TITLE
Fixed relative paths inside themed and localized files

### PR DESCRIPTION
**Pull request by SamMousa**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #14636

### CODE REVIEW

**Comment by samdark** [comment]
AppVeyor fail isn't connected with this pull request.

File: framework/base/View.php [hunk 1]
```diff
@@ -267,7 +270,15 @@ public function renderFile($viewFile, $params = [], $context = null)
...
     public function getViewFile()
     {
-        return end($this->_viewFiles);
+        return empty($this->_viewFiles) ? false : end($this->_viewFiles)['resolved'];
+    }
+
+    /**
+     * @return string|bool the requested view currently being rendered. False if no view file is being rendered.
+     */
+    public function getRequestedViewFile()
```
[the remark this answers is not in the document]
Comment (SamMousa) [reply framework/base/View.php]: Well it doesn't need to be for the implementation to work, but I thought it makes sense to keep it consistent with `getViewFile`

[the remark this answers is not in the document]
Comment (samdark) [reply framework/base/View.php]: Let's have it protected. Please add `@since`.

**Comment by samdark** [comment]
Changelog is needed.

**Comment by samdark** [comment]
Any idea why HHVM fails?

**Comment by SamMousa** [comment]
:( No idea.

**Comment by SamMousa** [comment]
@samdark I think this is ready :)

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**